lib/gds_api/email_alert_api.rb

```diff
@@ -105,8 +105,41 @@ def subscribe(subscribable_id:, address:, frequency: "immediately")
   #  subscriber_count
   # }
   def get_subscribable(reference:)
-    get_json(
-      "#{endpoint}/subscribables/#{reference}"
+    get_json("#{endpoint}/subscribables/#{reference}")
+  end
+
+  # Get Subscriptions for a Subscriber
+  # #
+  # @param string Subscriber address
+  #
+  # @return [Hash] subscriber, subscriptions
+  def get_subscriptions(address:)
+    get_json("#{endpoint}/subscribers/#{address}/subscriptions")
+  end
+
+  # Patch address for a Subscriber
+  # #
+  # @param string Subscriber old_address
+  # @param string Subscriber new_address
+  #
+  # @return [Hash] subscriber
+  def change_subscriber_address(old_address:, new_address:)
```

Comment:
I would call this `change_subscriber(address:, new_address:)` in case we add more fields to the `PATCH` endpoint in the future.

Reply:
👍